.travis.yml

```diff
@@ -72,10 +72,9 @@ matrix:
         - sudo ln -s $(which ccache) bin/arm-none-eabi-gcc
         - sudo ln -s $(which ccache) bin/arm-none-eabi-g++
         - export PATH="$(pwd)/bin:$PATH"
-        # Fetch mbed-os: We use manual clone, with depth=1 and --single-branch to save time.
-        - git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git
         # Install Mbed CLI and dependencies
         - pip install --upgrade mbed-tools
+        - mbed-tools deploy
```

Comment:
Should we make this change for CircleCI as well?

Reply:
I did so. This means Circle CI will use what's in `mbed-os.lib` rather than the latest `master` branch of Mbed OS, so it's a change of behavior (though maybe a welcome one).